src/Logary.Facade/Facade.fs

```diff
@@ -315,6 +457,113 @@ module internal Formatting =
     formatExn message.fields +
     formatFields matchedFields message.fields
 
+  let literateExceptionColorizer (context: LiterateContext) (ex: exn) =
+    let stackFrameLinePrefix = "   "
+    use exnLines = new System.IO.StringReader(ex.ToString())
+    let rec go lines =
+      match exnLines.ReadLine() with
+      | null -> lines // finished reading
+      | line -> if line.StartsWith(stackFrameLinePrefix) then
+                  go (lines @ [ line, Subtext; Environment.NewLine, Text ])
+                else
+                  go (lines @ [ line, Text; Environment.NewLine, Text ])
+    go []
+
+  let literateLogLevelTokenizer = function
```

Comment:
Ah, I see your point. Let me think about it, but if I don't comment again, feel free to keep the current version.